sdks/python/apache_beam/io/avroio_test.py

```diff
@@ -145,6 +146,29 @@ def test_read_with_splitting_pattern(self):
     expected_result = self.RECORDS * 3
     self._run_avro_test(pattern, 100, True, expected_result)
 
+  def test_read_split_points(self):
+    file_name = self._write_data(count=20)
+    unsplit_source = AvroSource(file_name)
+
+    # desired_bundle_size should be large enough to make sure that first split
+    # is not a trivial one.
+    splits = [split for split in unsplit_source.split(1000)]
+    if len(splits) < 1:
+      raise ValueError('Split of source %r did not produce any results',
+                       unsplit_source)
+
+    source = splits[0].source
+    start_position = splits[0].start_position
+    stop_position = splits[0].stop_position
+
+    range_tracker = source.get_range_tracker(start_position, stop_position)
+    assert isinstance(range_tracker, iobase.RangeTracker)
+    reader = source.read(range_tracker)
+    for _ in reader:
+      self.assertEquals((0, iobase.RangeTracker.SPLIT_POINTS_UNKNOWN),
+                        range_tracker.split_points())
```

Comment:
The entire point of remaining parallelism is that once you start reading the last Avro block you can tell it's the last Avro block. In other words, remaining parallelism should go `UNKNOWN`, then `1`, then `0`. If we only go `UNKNOWN` then `0` then remaining parallelism is useless.